script/Ciel/Day 5/05_01_CIEL05_4.txt

```diff
@@ -879,6 +879,7 @@ However, the pain carved into his soul could never be healed.
 -- 寒さという名の乾きには抗えなかった。
 // 乾き short form dict entry is 'dryness', but it can also be parched / thirsty.
 // Rendering as thirst here, since I think that's consistent with the cold being a thirst for warmth (blood)
+// The phrasing makes it sound like he's thirsting for cold - would it be too much to replace 'the cold' with 'warmth'?
```

Comment:
Well the thirst is the opposite of the thing thirsted for - thirst _for_ warmth would make sense (a bit of a departure though) but renaming the thirst / state of being thirsty to warmth here would change the meaning. The thirst _of being_ cold might be more clear?

Reply:
He could not resist the drought known as 'the cold'.